skimage/measure/_marching_cubes_lewiner_cy.pyx

```diff
@@ -929,13 +929,15 @@ cdef class LutProvider:
 
         self.SUBCONFIG13 = Lut(SUBCONFIG13)
 
-
-def marching_cubes(float [:, :, :] im not None, double isovalue, LutProvider luts, int st=1, int classic=0):
+def marching_cubes(float [:, :, :] im not None, double isovalue, LutProvider luts, int st=1, int classic=0,
```

Comment:
Can I ask you @Leguark for a last small modification before merging? Please wrap these few lines to 80 characters length.. Thank you in advance ;-)

Reply:
It was the optimization that @jni suggested in order to check is mask is None only once and not once per voxel. Regarding the wrap, sure no problem. I have it at 120 but if the whole project is at 80 makes sense

Reply:
Thank you for the code suggestions. Makes it much easier since I am on the road :)